### ISSUE TITLE
Batch Size and Adam Optimizer

### ISSUE THREAD

**Issue by rruixxu** [yangyanli/PointCNN#152]
Hi, 
I am reading your tensorflow code, but I found the batch size used in the code is different from that declared in your NIPS paper. In the code, batch size is 128 for ModelNet40, however, it becomes 16 in your paper. What is the real batch size?

I also found that epsilon in Adam optimizer is 0.01, but epsilon for Adam is usually very small like 1e-7 or 1e-8. The purpose of epsilon is to prevent from dividing by zero, isn't it? Is it possible or are there other motivations for you to set epsilon so large?

Thank you!

**Comment by burui11087** [comment]
Hi @rruixxu 
We  want to compare network complexity with other methods metioned in Table 6 in our NeurIPS paper, we set batch size equal to 16 for fair beacause some of them can't set a large batch size. 
That's to say, 16 is not our default batch size in our classification task network.

**Comment by burui11087** [comment]
@rruixxu 
Our Learning rate  is very large, so we set epsilon 0.01 to avoid network jitter when training.
